rewrite-xml/src/test/java/org/openrewrite/xml/trait/SpringReferenceTest.java

```diff
@@ -67,13 +70,16 @@ void xmlConfiguration() {
                   <property name="age" value="10"/>
                   <property name="sibling">
                       <bean <!--~~(org.springframework.beans.TestBean)~~>-->class="org.springframework.beans.TestBean">
-                          <property name="age" value="11" <!--~~(java.lang.Integer)~~>-->class="java.lang.Integer"/>
+                          <property name="age" value="11" <!--~~(java.lang.int)~~>-->class="java.lang.int"/>
```

Comment:
This one's a bit odd; was that intentional?

Reply:
Nope, accidentally left that in, good find!

Reply:
What is interesting is that we can't really add comments in front of attributes like this. It renders the XML invalid. But I suspect that is acceptable.

Reply:
The only purpose of the test is to verify the right elements are identified as type reference